aws/cluster.tf

```diff
@@ -76,18 +76,20 @@ resource "aws_instance" "redpanda" {
   instance_type              = var.instance_type
   key_name                   = aws_key_pair.ssh.key_name
   iam_instance_profile       = var.tiered_storage_enabled ? aws_iam_instance_profile.redpanda[0].name : null
-  vpc_security_group_ids     = concat([aws_security_group.node_sec_group.id], var.security_groups_redpanda)
   placement_group            = var.ha ? aws_placement_group.redpanda-pg[0].id : null
   placement_partition_number = var.ha ? (count.index % aws_placement_group.redpanda-pg[0].partition_count) + 1 : null
   availability_zone          = var.availability_zone[count.index % length(var.availability_zone)]
-  subnet_id                  = var.subnet_id
   tags = merge(
     local.merged_tags,
     {
       Name = "${local.deployment_id}-node-${count.index}",
     }
   )
 
+  vpc_security_group_ids      = concat([aws_security_group.node_sec_group.id], var.security_groups_redpanda)
```

Comment:
Slightly confused as to why this moved. As far as I can see there's no change?

Reply:
moved back